### DISCUSSION TITLE
Any support for multiple launchers?

### DISCUSSION THREAD

**Discussion by bogo09** [RareDevs/Rare#431] [Q&A]
I'd like to open 3 versions of the same game, I managed to open 2 with both regular Epic & Rare. Is there any way using Rare or maybe something else to open 3 versions? Thanks!

**Comment by loathingKernel** [comment]
I am unsure if that would work, because multiple launchers would use the same token for authentication credentials. You can try by initially using EGS, Rare and Legendary from the command line. If a came can launch multiple instances using Rare and Legendary at the same time, then it might be possible to implement something in Rare to achieve the same result.

**Reply by bogo09** [reply]
Ok! Im not the best at command lines but Ill see if I can make something work and will report back if I do! Tyvm!

**Reply by bogo09** [reply]
Worked wonders! Was able to open 3! Tyvm!
